l1-contracts/contracts/zksync/interfaces/IExecutor.sol

```diff
@@ -155,18 +155,29 @@ interface IExecutor is IBase {
     function revertBatches(uint256 _newLastBatch) external;
 
     /// @notice Event emitted when a batch is committed
+    /// @param batchNumber Number of the batch committed
+    /// @param batchHash Hash of the L2 batch
+    /// @param commitment Verified input for the zkSync circuit
```

Comment:
it is not yet verified to be correct at the commitment stage. I would rather use word "calculated" here